### ISSUE TITLE
Feature request: RSS add uploader

### ISSUE THREAD

**Issue by Jisagi** [nyaadevs/nyaa#450]
It would be nice to have the user who uploaded a torrent in the feed when not using &u=NAME. This would help filter out unwanted entries when parsing rather than sending a request for every user one wants to check for with a new request. Something like <nyaa:uploader/user> or <uploader/user>.

**Comment by Arylide** [comment]
It would probably be better to just have multiple user augments possible by stacking the `&u=`.

**Comment by Jisagi** [comment]
This might be a solution in some cases but not in all. If you want to just check for keywords and not regular releases by certain users, this won't help.

**Comment by CounterPillow** [comment]
You can make an RSS feed that filters for keywords and limits it to a user just fine right now.

**Comment by Jisagi** [comment]
I know that, but it's not what i meant. I meant being able to filter all new releases but "verify" a certain release is from a user, for example to prevent fakes triggering whatever parses the feed.
I can either filter all releases and hope something called "[group] animename[...].ext" is from the group it's filename says (which sometimes it isn't) or i parse the feed twice, one time for all users and one time for the user i think it should belong to and then compare if that release exists in both feeds, which would be imo kinda stupid. Having the user in the feed would would elimite every extra call to verify it's actaully from that user or parsing x amount of user feeds.
Please don't misunderstand me, i know it is possible to solve this the way both of you said. I just thought it would at least lower the amount of requests sent in certain not that far fetchted cases.

**Comment by Phr33d0m** [comment]
> and one time for the user i think it should belong to and then compare if that release exists in both feeds

Exist in both feeds? That makes literally no sense.

Just subscribe to all the feeds of the users you're interested in (and the general one if you want)

![image](https://user-images.githubusercontent.com/1591486/40143053-a3c5bc00-595a-11e8-83cd-eb305814f4c5.png)

Then just apply the showname filter to the appropriate feed:

![image](https://user-images.githubusercontent.com/1591486/40142776-ecdeccb6-5959-11e8-8916-bdb57e32a6cc.png)

**Comment by Jisagi** [comment]
@Phr33d0m 
I'll just repeat what i already said above: "Please don't misunderstand me, i know it is possible to solve this the way both of you said. I just thought it would at least lower the amount of requests sent in certain not that far fetchted cases."
I know it's already possible, but each new uploader one wants to check means a new feed to check and a request. If you now want to check uploads on a regular basis, like every day, hour, minute whatever just think of the additional load on the api if you check lets say 20+ uploads every other minute vs. just checking once for all of them. It's easier for the user and less load for the server.
That's why i recommended it, not as new feature, but as enhancement for the current solution.

**Comment by Phr33d0m** [comment]
@Jisagi I understood you the first time, but I should have spent more time explaining why it's useless. The main reason (I'll use qBittorrent as an example) is that torrents clients usually (if not only) filter just by the ```<title>``` attribute.

Yes, I understand that you might want to code it yourself and so on, although the use case is so small that I don't see it as such an improvement. Also, if there's any caching in-place - the improvement over the "less" API requests would be unnoticeable - mostly because:

1. The extremely low use of this use case (as I commented, torrents clients wouldn't be able to filter that new attribute at all)
2. Even if the rss is not cached (which I doubt it is, because of the many parameters it gets to generate the content (and yeah, I'm too lazy to check the sauce)) - the lowered API calls (that you mention) would be meaningless in comparision to the total amount of users using even 1 RSS.

**Comment by Jisagi** [comment]
First of all, it was an idea, not a "I WANT THIS NOW" thing. It awfully sounds like you just don't like by saying over and over again it's "useless", cool down. It was just a suggestion, nothing else. If the nobody/devs don't like, i'm ok with that.
Also, i doubt 99.9% uses the RSS only with a torrent client but some custom script, bot or whatever else. Saying your torrent client wouldn't be able to use that certain attribute for an RSS feed in XML is stupid. XML is made to be extensible. You can put whatever data you want into it.
I don't use the RSS inside some self made torrent client or anything like that, but as notification web-feed **which is the intended use of an RSS**. Nothing fancy just so say i did it myself. I doubt i am the only one using it for this purpose.

TL;DR: if nobody likes, then just close it. it was just an idea ¯\_(ツ)_/¯
